api/plot/plotserver.py

```diff
@@ -0,0 +1,33 @@
+from __future__ import annotations
+
+import msgpack
+
+from typing import Any
+
+from plot.processor import Processor
+
+class PlotServer:
+    def __init__(self, processor: Processor):
+        self.ws_list = []
+        self.processor = processor
+        self.react_status = 'busy'
+        self.response_list = []
+        self.initialise_data()
+
+    def initialise_data(self):
+        for data in self.processor.initial_data:
+            msg = msgpack.packb(data, use_bin_type=True)
+            self.response_list.append(msg)
+
+    async def send_next_message(self):
+        self.react_status = 'ready'
+        if len(self.response_list) > 0 and len(self.ws_list) > 0:
+            assert(self.ws_list[0])
+            assert(self.response_list[0])
+            await self.ws_list[0].send_text(self.response_list.pop(0))
```

Comment:
Need a method to wait for response from the client and update `react_status` (better as `client_status`